Resources/Prototypes/_Goobstation/Entities/Clothing/Neck/specific.yml

```
@@ -0,0 +1,32 @@
+    transmitFrequencyId: SurveillanceCamera
+  - type: SurveillanceCamera
+    networkSet: true
+    nameSet: True
+    id: Bodycam
+  - type: Eye
+  - type: WirelessNetworkConnection
+    range: 200
+  - type: SurveillanceCameraMicrophone
+    blacklist:
+      components:
+      - SurveillanceCamera
```

yamllint (1.35.1)

[error] wrong indentation: expected 8 but found 6

(indentation)